Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -237,6 +237,15 @@ input[type=submit], input[type=button], input[type=reset], input[type=checkbox],
     cursor: unset;
 }
 
+input[type=button] {
+    padding: 2px 8px;
+    background-color: #d4d0c8;
+    border: 1px solid #808080;
+}
+input[type=button]:hover {
+    background-color: #e3dfdb;
+}
```

Comment:
Can we use `-libweb-palette-*` colors here instead?

Reply:
Adjusted this.